docs/fr-FR/_summary.md

```diff
@@ -0,0 +1,125 @@
+  * [Keymap / Disposition](faq_keymap.md)
+  * [Installer les drivers avec Zadig](driver_installation_zadig.md)
+
+* Guides détaillés
+  * [Installation des outils de compilation](getting_started_build_tools.md)
+  * [Guide Vagrant](getting_started_vagrant.md)
+  * [Commandes de compilations](getting_started_make_guide.md)
+  * [Flasher les firmwares](fr-fr/flashing.md)
+  * [Customiser les fonctionnalités](custom_quantum_functions.md)
+  * [Apercu des fonctionnalités des dispositions](keymap.md)
+
+* [Hardware](hardware.md)
+  * [Processeurs AVR](hardware_avr.md)
+  * [Pilotes / Drivers](hardware_drivers.md)
+
+* Réferences
+  * [Lignes de conduite des claviers](hardware_keyboard_guidelines.md)
+  * [Options de configurations](config_options.md)
+  * [Keycodes / Codes des caractères](keycodes.md)
+  * [Conventions de codage - C](coding_conventions_c.md)
+  * [Conventions de codage - Python](coding_conventions_python.md)
+  * [Meilleurs pratiques sur la documentation](documentation_best_practices.md)
+  * [Modèles de documentation](documentation_templates.md)
+  * [Glossaire](reference_glossary.md)
+  * [Tests unitaires](unit_testing.md)
+  * [Fonctions utiles](ref_functions.md)
+  * [Support de configuration](reference_configurator_support.md)
+  * [Format du fichier info.json](reference_info_json.md)
+  * [Développer du Python en ligne de commande](cli_development.md)
+
+* [Fonctionnalités](features.md)
+  * [Keycodes basiques](keycodes_basic.md)
+  * [Touches utilisées avec Shift (US ANSI)](keycodes_us_ansi_shifted.md)
+  * [Keycodes quantiques](quantum_keycodes.md)
+  * [Keycodes avancés](feature_advanced_keycodes.md)
+  * [Fonctionnalités audio](feature_audio.md)
+  * [Majuscule automatique](feature_auto_shift.md)
+  * [Rétroéclairage](feature_backlight.md)
+  * [Bluetooth](feature_bluetooth.md)
+  * [Bootmagic](feature_bootmagic.md)
+  * [Combos](feature_combo.md)
+  * [Commande](feature_command.md)
+  * [API anti-rebond](feature_debounce_type.md)
+  * [DIP Switch](feature_dip_switch.md)
+  * [Macros dynamiques](feature_dynamic_macros.md)
+  * [Interrupteurs rotatifs](feature_encoders.md)
+  * [Grave Escape](feature_grave_esc.md)
+  * [Retour haptique](feature_haptic_feedback.md)
+  * [Contrôleur LCD HD44780](feature_hd44780.md)
+  * [Touche à verrou / Lock-key](feature_key_lock.md)
+  * [Dispositions / layouts](feature_layouts.md)
+  * [Touche leader](feature_leader_key.md)
+  * [Matrice LED](feature_led_matrix.md)
+  * [Macros](feature_macros.md)
+  * [Boutons de souris](feature_mouse_keys.md)
+  * [Pilotes / Drivers OLED](feature_oled_driver.md)
+  * [Touche one-shot](feature_advanced_keycodes.md#one-shot-keys)
+  * [Périphériques de pointage](feature_pointing_device.md)
+  * [Souris PS/2](feature_ps2_mouse.md)
+  * [Éclairage RGB](feature_rgblight.md)
+  * [Matrice RGB](feature_rgb_matrix.md)
+  * [Space Cadet](feature_space_cadet.md)
+  * [Claviers scindés / splittés](feature_split_keyboard.md)
```

Comment:
`scindés` is never used in french. `Split` may be more accurate

Reply:
LOL, we had a debate about this with @4sStylZ it was the best translation in French we could find... There was this discussion on the BEPO forum ttp://forum.bepo.fr/viewtopic.php?id=1505.

We kind of liked scindé in the end 😄. I mean, at least it's French.

Reply:
What's your opinion on this?

Reply:
I have no real opinion, but i prefer `split` as it's the more common used term.

Reply:
Go for split :) it's a english word but at least it's the good word.